www/pages/docs/build.md

```diff
@@ -0,0 +1,36 @@
+---
+label: 'build'
+menu: side
+title: 'Build'
+index: 4
+linkheadings: 3
+---
+
+# Build Configurations
+
+A number of core build configuration files can be overridden by creating each file within the root path of your project. You can also automate this task in a single command [see eject configurations](#eject-configurations).
```

Comment:
Maybe a link to the [configuration files](https://github.com/ProjectEvergreen/greenwood/tree/master/packages/cli/src/config) in the repo?